dependent/bundesgesundheitsblatt-gesundheitsforschung-gesundheitsschutz.csl

```diff
@@ -1,15 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="de-DE">
-  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer-fachzeitschriften-medizin -->
+  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer -->
   <info>
-    <title>Bundesgesundheitsblatt - Gesundheitsforschung - Gesundheitsschutz (German)</title>
-    <title-short>Bundesgesundheitsblatt Gesundheitsforschung Gesundheitsschutz</title-short>
+    <title>Bundesgesundheitsblatt - Gesundheitsforschung - Gesundheitsschutz</title>
```

Comment:
Ah, I see. In these examples the issn changed to an eissn and therefore the lookup is not working. But we could make the lookup for matching either issn or eissn in the excel sheet. Then we shouldn't loose the short-title data. Should we do that?